src/cryptoadvance/specter/static/styles.css

```diff
@@ -1192,6 +1204,9 @@ nav.collapse-on-mobile a.mobile-nav-icon {
     .main-page-container{
         background-image: none;
       }
+    .mobile-distance-element-botom {
+        height: var(--mobileDistanceElementBotomHeight, mobileDistanceElementBotomHeight);
```

Comment:
Doesn't make sense to repeat the variable here, better to provide some px value as fallback.

<img width="676" alt="grafik" src="https://user-images.githubusercontent.com/70536101/183475079-058ffc71-cf7a-41aa-bb84-46ecb615b1d2.png">

Reply:
Thanks for the reminder. Done.